### PR TITLE
Adds Turbans, Kippot, and A Hoodie

### PR DESCRIPTION
**Pull request by redstryker**
Adds the following to the loadout:

- Kippot (in all colors)

- Turbans (in all colors as well)

- Space Mountain Wind Hoodie

Also fixes some of the code with the Hijabs loadout.  It is now cleaner.  Instead of being listed as (for example) 'Black Hijab' on the loadout screen, it is listed as 'Hijab, black'.  This allows it to be more orderly.

### CODE REVIEW

**Comment by Mark9013100** [comment]
does the hoodie come with doritos

**Comment by Spacemanspark** [comment]
@Mark9013100 Don't forget air horns.

**Comment by Mark9013100** [comment]
:honk:

**Comment by zaredman** [comment]
https://www.youtube.com/watch?v=nlxfDvSyPKA

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -77,7 +77,7 @@
 	display_name = "cap, orange"
 	path = /obj/item/clothing/head/soft/orange
 
-/datum/gear/head/cap/purple
+/datum/gear/head/cap/orange
```
Comment (PsiOmegaDelta) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: The purple hat has became orange, once again.

**Comment by Yoshax** [comment]
Screenshots?

**Comment by redstryker** [comment]
Kippa (in red):
http://puu.sh/mWkK6/541fe8a5f2.png
http://puu.sh/nCT6r/0e9f98eb20.png
http://puu.sh/nCT7W/2de785eea2.png
No front view since it cannot be seen from the 'north' position.

Turban (in red):
http://puu.sh/nCSOh/aa96aa764b.png
http://puu.sh/nCSS9/111bfeb905.png
http://puu.sh/nCSXQ/a1bf497524.png
http://puu.sh/nCT0G/7d9e33e420.png

Hoodie:
http://puu.sh/mm13w/e19ed2af1c.png (front's pretty plain excluding like, four pixels)

**Comment by Kelenius** [comment]
I don't see any issues with the code, but you really need to learn to use git properly :<

**Comment by PsiOmegaDelta** [comment]
The fact that you begin with deleting whole files only to replace them again is a bit concerning.

**Comment by redstryker** [comment]
I'm working on it.  Keep in mind this is my third PR, so give me some credit.

**Comment by EmperorJon** [comment]
Marking as DNM, basically because of what Psi said. That's a lot of questionable commits.

**Comment by Kelenius** [comment]
@redstryker You won't learn to use git properly by using it improperly.

**Pull request closed without merging**